index.js

```diff
@@ -64,25 +47,7 @@ module.exports = function ({ content, filename, sourceMap }) {
       + transpile(`${exportsTarget}.staticRenderFns = [${staticRenderFns}];`)
       + `\n${exportsTarget}.render._withStripped = true;`;
   }
-  const result = `${scriptPart}${compiledTemplate}`;
```

Comment:
Yes it should be fine (in theory). So a component like:
```html
<template>
<div><slot></slot></div>
</template>
<style>
div{}
</style>
<script>
export default {}
</script>
```
will come out like
```html
//
//
//
//
//
//
//
exports.default = {}
exports.default.render = ...
```
something like that. So debugging should be kept intact. I might debug a few components like this to check it's right and then I'll merge it